odyssy-automaton/pocket-moloch
# Chore/fix web3 version and int parsing

## What does it do?
This PR pins the web3 version to `beta34` in order to ensure that all browsers can use web3 when using infura. Otherwise CORS issues will arise.

It also changes the way numbers are parsed when returned as strings from web3. It seems to be returning strings which need a different parser, `parseInt`.

## How to test?
```
yarn start
```
* Open on firefox and chrome. (Feel free to check safari if on mac or edge if on windows).
* See that everything works.
